signup.html

```diff
@@ -1,54 +1,83 @@
 <html>
-    <head>
-        <link rel="stylesheet" type="text/css" href="style.css" /> 
-        <title>EMS- Signup</title>
-        <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
-    </head>
-    <body>
-    <div class="nav h-16">
-    <a href="index.html">EMPLOYEE MANAGEMENT SYSTEM  </a>  
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>EMS- Signup</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <style type="text/tailwindcss">
+      @layer utilities {
+  a {
+    color: rgb(193, 255, 255);
+  }
+  body {
+    background-color: rgb(177, 185, 192);
+  }
+  label {
+    color: rgb(180, 180, 180);
+    font-size: large;
+  }
+  input {
+    border: none;
+    padding: 4px 10px;
+    width: 300px;
+    font-size: large;
+  }
+  .error {
+    color: rgb(250, 134, 134);
+  }
+}
+    </style>
+    
+  </head>
+  <body>
+    <div class=" h-16 bg-red-900 text-white flex justify-between p-2 items-center font-extralight">
+      <a href="index.html">EMPLOYEE MANAGEMENT SYSTEM </a>
     </div>
-   
-    <div class="title">Signup</div>
 
-    <form class="form">
-        <div class="form-field ">
-        <label>Name</label><br>
-        <input  type="text" required name="name"></input>
+    <div class="mt-5 flex text-xl justify-center font-bold">Signup</div>
+
+    <form class="mx-auto my-3 p-5 bg-[#003741] rounded-2xl flex flex-col items-center w-[500px]">
+      <div class="m-auto w-auto mt-3 mb-4 text-white">
+        <label>Name</label><br />
+        <input placeholder="Enter you full name" type="text" required name="name" />
         <div class="error" id="error-name"></div>
-        </div>
-        <div class="form-field">
-            <label>DOB</label><br>
-            <input type="date" required name="dob"></input>
-            <div class="error" id="error-dob"></div>
-            </div>
-        <div class="form-field">
-        <label>Email</label><br>
-        <input type="email" required name="email"></input>
+      </div>
+      <div class="m-auto w-auto mt-3 mb-4 text-white">
+        <label>DOB</label><br />
+        <input placeholder="enter your date of birth" type="date" required name="dob" />
```

Comment:
```suggestion
        <input placeholder="Enter your date of birth" type="date" required name="dob" />
```
Placeholders or Labels first character should be capital only.

Reply:
fixed